src/storage/v3/shard_manager.hpp

```diff
@@ -212,6 +215,17 @@ class ShardManager {
 
     std::unique_ptr<Shard> shard =
         std::make_unique<Shard>(to_init.label_id, to_init.min_key, to_init.max_key, to_init.schema, to_init.config);
+    // TODO Should be sync
```

Comment:
Could you please blame the file and put a username for this TODO? As the comment is not very telling, it is good to remember who put this TODO here.